### PR TITLE
store: removed duplicate method call for the same method

### PR DESCRIPTION
**Pull request by sinsharat**
the get func was calling path's Join and clean method which is already
being in internalGet(nodePath) func. Hence the func was getting called
unnecessarily twice which is not needed.

### CODE REVIEW

File: wal/doc.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ to it with the Save method:
 	...
 	err := w.Save(s, ents)
 
-After saving an raft snapshot to disk, SaveSnapshot method should be called to
+After saving a raft snapshot to disk, SaveSnapshot method should be called to
```
Comment (xiang90) [inline wal/doc.go]: can we separate this into another pr?

**Comment by sinsharat** [comment]
@xiang90 sure will do it.
